### PR TITLE
Made project report changes

### PR DESCRIPTION
**Pull request by shriram-sg**
1. Changed about page to include details about python first.
2. Removed explanation from softwares used page and moved school syllabus related items up.
3. Removed feasibility study subsections from ToC

### CODE REVIEW

File: latex-report/main.tex [hunk 1]
```diff
@@ -188,35 +188,35 @@
...
   The aim of the project, Sudoku Webapp, is to provide an aesthetically rich interface for sudoku enthusiasts to enjoy sudoku puzzles.\newline
 
-  A website with 3 webpages: home page, sudoku page and leaderboard page, is the frontend of the project with which the user interacts. The webpages are made using Django, HTML, JavaScript and a CSS framework called Tailwind. A MySQL database is the primary data store, and is used to store the leaderboard data, and the sudoku game boards that are served on the site.\newline
+  A website with 3 webpages - home page, sudoku page and leaderboard page - is the frontend of the project with which the user interacts. The entire backend is handled using python and the django framework. The sudoku game boards are generated using a combination of custom classes and functions as well as functions from the random module in python. These game boards are passed onto the website using queues. A MySQL database is the primary data store which is used to store the leaderboard data, and the sudoku game boards that are served onto the site. The webpages are made using HTML, JavaScript and a CSS framework called TailwindCSS. \newline
```
Comment (anand2312) [inline latex-report/main.tex]: Capitalize `Python` and `Django` here I'd say; and if possible everywhere else they're used. (Not very high priority though)

File: latex-report/main.tex [hunk 2]
```diff
@@ -641,14 +641,14 @@
                 return element
             count += 1
 
-    def __setitem__(self, index: int, val: typing.Union[int, None]):  # noqa: ANN204
+    def __setitem__(self, index: int, val: typing.Optional[int]=None):  # noqa: ANN204
```
Comment (anand2312) [inline latex-report/main.tex]: Nice, using the Optional type hint is better 👍🏻 
If you do end up editing this file again, add [spaces around the `=` for the default](https://www.python.org/dev/peps/pep-3107/#parameters) (Note that if there are no type-hints, no space is expected there)

**Review by anand2312 — APPROVED** [review]
LGTM, you can decide if you want to make these changes (rather minor, so I don't mind if you don't)

File: latex-report/main.tex (hunk 1 above)
Comment (shriram-sg) [reply latex-report/main.tex]: Done

File: latex-report/main.tex (hunk 2 above)
Comment (shriram-sg) [reply latex-report/main.tex]: I'll make the change, but lets not print it out.

**Review by anand2312 — APPROVED** [review]
LGTM, merge it 👍

**Pull request merged**